### ISSUE TITLE
ng add throws error (An unhandled exception occurred: Cannot find module 'angular-cli-ghpages/ng-add')

### ISSUE THREAD

**Issue by fmalcher** [angular-schule/angular-cli-ghpages#156]
This is due to a mistake during publishing of 1.0.1. We will fix this asap.

**Comment by JohannesHoppe** [comment]
I'm on it

**Comment by JohannesHoppe** [comment]
fixed via 1.0.3

The command line arguments of NPM 8 have been changed!

https://docs.npmjs.com/cli/v7/commands/npm-publish
```
npm publish [<tarball>|<folder>] [--tag <tag>] [--access <public|restricted>] [--otp otpcode] [--dry-run]
```

https://docs.npmjs.com/cli/v8/commands/npm-publish
```
npm publish <package-spec>
```

**Comment by JohannesHoppe** [comment]
```
npm deprecate -f 'angular-cli-ghpages@1.0.1' "this package is broken, please use v1.0.3"
npm deprecate -f 'angular-cli-ghpages@1.0.2' "this package is broken, please use v1.0.3"
```
